src/MusicalScore/VoiceData/CustomChordKind.ts

```diff
@@ -0,0 +1,45 @@
+import { ChordSymbolEnum } from "./ChordSymbolContainer";
+
+export class CustomChordKind {
+  public alternateName: string;
+  public chordKind: ChordSymbolEnum;
+  public adds: string[];
+  public alts: string[];
+  public subs: string[];
```

Comment:
of these class members, adds, alts and subs are currently unused. are they just informational?

Reply:
@sschmidTU - this is the only one I'm stumped on...
These are the arrays of degrees for each degree-type.  Is there another way I should do this?

Reply:
Hmm, not sure on a first look, but if they are relevant in any way, shouldn't they be used somewhere? Like for `addCustomChordKind()` or the renaming of chords (give adds, alts, subs as arguments)?

Reply:
Yeah, addCustomChordKind takes five arguments... three of which are the adds, alts, and subs...

I'm also looking at your earlier comment about renaming the class... or even integrating CustomChordKind in the existing ChordSymbolContainer.  If that's your inclination, I want to give that a shot!  That might make this a moot point.